src/fragments/lib/in-app-messaging/js/create-campaigns.mdx

```diff
@@ -0,0 +1,56 @@
+3. Click on “Campaigns” from the left navigation menu, and then click on “Create a campaign”
+
+   ![In-App-Messaging](/images/lib/in-app-messaging/create-pinpoint-campaign.png)
+
+4. Add a name to your campaign, and keep the following options as follows and then click Next:
+
+   1. Campaign type: **Standard campaign**
+   2. Channel: **In-App messaging**
+   3. set prioritization: **Fairly important**
+
+5. Click on the “Create a segment” radio button, and add a name for your segment, and then click on Next.
+
+   1. You can add as many segments as needed to the campaign. For this quickstart, you can use “Include any audiences” under the “Segment group 1” section.
+   2. You can add a criteria to your segments to ensure that audiences that satisfy that criteria can receive the in-app message.
+   3. If you see an error message titled “Segment might include multiple channels”, click on “I understand” to proceed.
+
+   ![In-App-Messaging](/images/lib/in-app-messaging/choose-campaign-segment.png)
+
+6. Click on the “Create a new in-app message” radio button.
+7. You have the ability to customize the following attributes of the in-app message:
+   - **Layout**: Which includes all of the different messaging layout options.
+   - **Header**: Title of the in-app message, including the text color/alignment.
+   - **Message**: The body of the Message, including the text color/alignment.
+   - **Background**: Control the background color of the in-app message.
+   - **Image URL**: Add an image to be displayed as part of the in-app message body.
+   - **Primary button**: Allows the addition of a button to add functionality to the in-app message.
+   - **Secondary button**: Allows the addition of an extra button for additional functionality.
+   - **Custom Data**: Allows the in-app message to pass additional data to the frontend app once it is triggered by an event.
+
+   <Callout informational>
+      As React Native does not support SVG rendering out of the box, Amplify cannot render SVG images by default. For SVG image support with In-App Messaging a custom UI implementation is required.
+   </Callout>
+
+For this tutorial you can create a simple message as shown below. Customers in your application will see the same message once it’s triggered on the client side.
+
+![In-App-Messaging](/images/lib/in-app-messaging/in-app-message-details.png)
+
+8. Once you have finished customizing your in-app message, click on “Next”.
+9. Under “Trigger events”, add the name of the analytics trigger that will be sent from your frontend app. 
+   - You have the ability to customize the trigger to allow only certain “attributes” or “metrics” that are passed with the analytics event to trigger the in-app message. (Optional)
+
+![In-App-Messaging](/images/lib/in-app-messaging/campaign-setup.png)
+
```

Comment:
A note telling the default message cap per session is 1 should be helpful to users. [They need to enter 0 for unlimited messages per session]

If a campaign is created as is, the test app receives only 1 in-app message per session and would be hard to diagnose why if they are not aware of this note.

Reply:
Thanks @Samaritan1011001 , I have added this information as a step with an image.